scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/InfoTreeGenerator.java

```diff
@@ -68,13 +70,11 @@ static InfoTreeGenerator findSuitableGenerator(String signature,
 	{
 		List<InfoTreeGenerator> suitableGenerators = generators.stream() //
 			.filter(g -> g.canGenerate(signature)) //
+			.sorted() //
 			.collect(Collectors.toList());
-		if (suitableGenerators.size() == 0) throw new IllegalArgumentException(
+		if (suitableGenerators.isEmpty()) throw new IllegalArgumentException(
 			"No InfoTreeGenerator in given collection " + generators +
 				" is able to reify signature " + signature);
-		if (suitableGenerators.size() > 1) throw new IllegalArgumentException(
-			"Signature " + signature +
-				" is able to be reified by multiple generators: " + suitableGenerators);
 		return suitableGenerators.get(0);
```

Comment:
What if the top two `InfoTreeGenerator` instances have the same priority? Should we fail fast? (I don't have an opinion, just pointing out that right now, on ties, one of them arbitrarily wins.)

Reply:
I'm going to say YAGNI for now